lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -108,7 +108,7 @@ def define
 
         next_node do |response|
           if response == "none"
-            outcome :housing_benefit_amount?
+            question :housing_benefit_amount?
```

Comment:
This change seems unrelated to the commit note. I think it belongs in a separate commit.